app/models/concerns/package_filtering.rb

```diff
@@ -109,26 +107,22 @@ def filter_by_location(location_name)
       end
     end
 
-    def join_order_packages
-      joins("LEFT OUTER JOIN orders_packages ON orders_packages.package_id = packages.id")
-    end
-
     private
 
     def received_sql
       "packages.state = 'received'"
     end
 
     def in_stock_sql
-      "packages.state = 'received' AND packages.quantity > 0"
+      "packages.state = 'received' AND packages.available_quantity > 0"
```

Comment:
Do we need to check for `state='received'`?

Reply:
@steveyken I don't really know, I figured I would just leave it as it was